internal/gcs/bridge.go

```diff
@@ -365,6 +367,7 @@ func (brdg *bridge) recvLoop() error {
 func (brdg *bridge) sendLoop() {
 	var buf bytes.Buffer
 	enc := json.NewEncoder(&buf)
+	enc.SetEscapeHTML(false)
```

Comment:
Why did we add this?

Reply:
default behavior is to escape characters for HTML, and since the replacement is `<scrubbed>`, that gets encoded as `\u003cscrubbed\u0033`, which i felt would cause issues for pattern matching down the line